### DISCUSSION TITLE
[Feature request] Differentiate toggle to activate invert title colours for full folder in light and dark themes

### DISCUSSION THREAD

**Discussion by smna-eeallatonce** [AnubisNekhet/AnuPpuccin#136] [Ideas]
I use the full folder feature, but there is not an option to differentiate the "invert title colours" option for the dark and light themes, which gets quite frustrating.

The theme of my obsidian adapts to my system settings, which means that I get the light theme in the morning and dark in the evening. I use both, and so I like the theme to be set up in a consistent way for both, without having to change it automatically every time, which is tedious and unproductive!

There is a similar feature for the "colourful frame" feature, so I imagine something similar where two separate toggle can be activate or deactivated could be feasible also for the full folder feature?

**Comment by AnubisNekhet** [comment]
Implemented with [this commit](https://github.com/AnubisNekhet/AnuPpuccin/commit/4fa11359b3ff55706e9dee85708690915cf00acd). I'd prefer if you used the Issues page for feature requests next time for the sake of convenience! 😁
